internal/cli/prompts_custom_text.go

```diff
@@ -214,9 +214,12 @@ func downloadDefaultBrandingTextTranslations(prompt, language string) map[string
 			return nil
 		}
 
-		selectedPrompt := allPrompts[0][prompt].(map[string]interface{})
```

Comment:
The issue is that the `allPrompts` is a set of twenty-something values, each with a single-property object pertaining to a specific prompt type.

This is why the `login` prompt was working, because it was the zeroth index in this set.